react/src/components/organisms/TabContainer/index.js

```diff
@@ -34,16 +63,36 @@ class TabContainer extends React.Component {
     const active = defaultTab;
     this.childrenWithProps = React.Children.map(children, (child, index) => {
       if (index === active) {
-        return React.cloneElement(child, { default: true });
+        return React.cloneElement(child, { default: true, a11yID: this.props.a11yID });
       }
       return child;
     });
+
     return(
       <TabContext.Provider value={this.state}>
         <div className={classes}>
-          <div className="ma__tab-container-head">{this.childrenWithProps}
-          </div>
-          {this.state.activeTab && <div className="ma__tab-container-body">{this.state.activeContent}</div>}
+          <span id={this.props.a11yID} className="ma__visually-hidden">Use left and right arrows to navigate between tabs, up and down arrows to navigate between active tab and its content.</span>
+          <ul className="ma__tab-container-head" role="tablist"
+              >{this.childrenWithProps}
+          </ul>
+
+
+
+// ADD onKeyDown to move the focus back to its paired tab. --> onKeyDown={this.handleTabContainerKeyDown}
+
+          {this.state.activeTab && <div className="ma__tab-container-body"
+                                    aria-labelledby={this.state.activeTab}
+                                    id={`tabpanel-${this.state.activeTab}`}
+                                    tabIndex="-1"
+                                    role="tabpanel"
```

Comment:
I don't think I agree with this being a tab panel. The container body may or may not be another listing of tabs, but it depends on how the container is being used. Maybe a better role that's more generic fits here?

Reply:
This is a common practice to set up tabs:  https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/tabs/tabs-2/tabs.html

aria roles are pre-defined:  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques
So, even it sounds unfitting, we cannot change it.

> ARIA tabs consist of a role="tablist" element, which has some some role="tab" elements. These tabs are then linked to role="tabpanel" elements that contain the corresponding content for each tab.
https://simplyaccessible.com/article/danger-aria-tabs/